tests/test_mpi.py

```diff
@@ -1357,7 +1446,7 @@ def test_merge_haloupdate_if_diff_locindices_v1(self, mode):
         op = Operator(eqns)
 
         calls = FindNodes(Call).visit(op)
-        assert len(calls) == 1
+        assert len(calls) == 2
```

Comment:
improved as discussed in : https://devitocodes.slack.com/archives/C7JMLMSG0/p1729152715068149